caluma/tests/__snapshots__/test_schema.ambr

```diff
@@ -563,7 +581,7 @@
     slug: String!
     label: String!
     document: Document!
-    question: StaticQuestion!
+    question: CalculatedFloatQuestion!
```

Comment:
It's a mystery to me what's going on with the question here. Even the previous value `StaticQuestion` is non-sense IMO.
It seems like this is tied to the order of declaration in schema.py, where `CalculatedFloatQuestion` is next after `StaticQuestion`.

Reply:
Anyway this is not in the scope of this pr